content/applications/finance/accounting/others/analytic/analytic_accounting.rst

```diff
@@ -0,0 +1,122 @@
+    confirm the entry if no analytic account is selected;
+  - :guilabel:`Unavailable`: if selected, the plan is not available in the widget.
+
+Two smart buttons are available in the top-right corner:
+
+   - :guilabel:`Subplans`: can be created to have a more complex analytic structure. Click the
+     :guilabel:`Subplans` smart button, and then :guilabel:`New` to add a subplan;
+   - :guilabel:`Analytic Accounts`: to reach the subplans related to the plan.
+
+.. note::
+   - The analytic widget is prefilled based on the :guilabel:`Applicability`, and the
+     :ref:`Analytic Distribution Models <analytic_distribution_models>`;
+   - Each :guilabel:`Analytic plan` must have at least one :guilabel:`Analytic Account`.
+
+Analytic distribution
+=====================
+
+When creating an invoice or a bill it is mandatory to add a plan in the
+:guilabel:`Analytic Distribution` column. This opens a **widget**, in which you can fill in the
+different information. You can add **tags** to reflect the related :guilabel:`Analytic Accounts`,
+and decide how to split the costs between the accounts by modifying the percentage.
+
+.. image:: analytic_accounting/analytic_distribution.png
+   :align: center
+   :alt: create a distribution template
+
+.. _analytic_distribution_models:
+
+Analytic distribution models
+----------------------------
+
+The :guilabel:`Analytic Distribution Models` automatically apply a specific distribution based on
+defined criteria.
+
+To create a new :guilabel:`Analytic Distribution Model`, go to :menuselection:`Accounting -->
+Configuration --> Analytic Distribution Models`, click :guilabel:`New` and set the conditions your
+model has to meet to automatically apply:
+
+- :guilabel:`Partner`: select a partner for which the analytic distribution will be used;
+- :guilabel:`Partner Category`: select a partner category for which the analytic distribution will
+  be used;
+- :guilabel:`Accounts Prefix`: this analytic distribution will apply to all financial accounts
+  sharing the prefix specified;
+- :guilabel:`Product`: select a product for which the analytic distribution will be used;
+- :guilabel:`Product Category`: select a product category for which the analytic distribution will
+  be used;
+  :guilabel:`Analytic Account` specified in analytic default;
+- :guilabel:`Company`: select a company for which the analytic distribution will be used;
+- :guilabel:`Analytic Distribution`: if the above conditions are met, the :guilabel:`Analytic plan`
+  defined in this field as well as the distribution to be applied between the different analytic
+  accounts is selected automatically on the entry.
+
+.. tip::
+   To **mass edit** several entries simultaneously, go to :menuselection:`Accounting --> Accounting
+   --> Journal items`, and select the ones that need to be updated. Add the required distribution in
+   the :guilabel:`Analytic Distribution` column, and click on the :guilabel:`diskette` icon to save.
```

Comment:
Again, just "Analytic" now it seems.
Floppy disk, not diskette 🤣

Reply:
it does exist! 
![image](https://user-images.githubusercontent.com/85342080/218485216-26daa14a-4148-478c-a6b6-91696f3e3412.png)